src/main/java/MainProgram.java

```diff
@@ -0,0 +1,12 @@
+import java.io.FileNotFoundException;
+
+public class MainProgram {
+  public static void main(String[] args) throws FileNotFoundException {
+    String path = "/home/gabriel/simbiose/Challenge-01/mlb_players.csv";
```

Comment:
This should not be harded-coded, you should receive this value as a program argument

Reply:
Resolved.